<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Kaiqian Zhang" />


<title>Abstract Summary for Barbara Engelhardt</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">paper-reading</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Abstract Summary for Barbara Engelhardt</h1>
<h4 class="author">Kaiqian Zhang</h4>
<h4 class="date">8/11/2019</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-08-16
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>paper-reading/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.4.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190301code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190301)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190301code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190301)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomKaiqianZhangpaperreadingtree7846b46f31df5811157e78a45d92a4123eb7f6actargetblank7846b46a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/KaiqianZhang/paper-reading/tree/7846b46f31df5811157e78a45d92a4123eb7f6ac" target="_blank">7846b46</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomKaiqianZhangpaperreadingtree7846b46f31df5811157e78a45d92a4123eb7f6actargetblank7846b46a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  analysis/QCBpaper17.Rmd

Unstaged changes:
    Modified:   analysis/QCBpaper20.Rmd
    Modified:   analysis/abstract-John.Rmd
    Modified:   analysis/basenji.Rmd
    Modified:   analysis/basset.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/noise-fate.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KaiqianZhang/paper-reading/blob/7846b46f31df5811157e78a45d92a4123eb7f6ac/analysis/abstract-barbara.Rmd" target="_blank">7846b46</a>
</td>
<td>
kaiqian
</td>
<td>
2019-08-16
</td>
<td>
update barbara notes again again.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KaiqianZhang/paper-reading/a20b7079437f32314b563ce93806c0521c0bc668/docs/abstract-barbara.html" target="_blank">a20b707</a>
</td>
<td>
kaiqian
</td>
<td>
2019-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KaiqianZhang/paper-reading/blob/8c1e77f25c7b9e48e14eb34a6576323a1fc6b6ac/analysis/abstract-barbara.Rmd" target="_blank">8c1e77f</a>
</td>
<td>
kaiqian
</td>
<td>
2019-08-14
</td>
<td>
update barbara notes again.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KaiqianZhang/paper-reading/1f0f093d8eb3280a27048d7aadbf9d656409ff32/docs/abstract-barbara.html" target="_blank">1f0f093</a>
</td>
<td>
kaiqian
</td>
<td>
2019-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KaiqianZhang/paper-reading/blob/cfab6588f9afc9b9dcb95022084ffc72c63bb1e4/analysis/abstract-barbara.Rmd" target="_blank">cfab658</a>
</td>
<td>
kaiqian
</td>
<td>
2019-08-14
</td>
<td>
update barbara notes.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KaiqianZhang/paper-reading/2127eaf99b2736df1871caec3c99759cc7c762b4/docs/abstract-barbara.html" target="_blank">2127eaf</a>
</td>
<td>
kaiqian
</td>
<td>
2019-08-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KaiqianZhang/paper-reading/blob/125df985cde1fc70a71d286f21013143d23510ce/analysis/abstract-barbara.Rmd" target="_blank">125df98</a>
</td>
<td>
kaiqian
</td>
<td>
2019-08-11
</td>
<td>
abstracts
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="cca" class="section level2">
<h2>CCA</h2>
<div id="deep-probablistic-cca" class="section level3">
<h3><code>Deep Probablistic CCA</code> :&gt;</h3>
<ul>
<li><p>Paper name: End-to-end training of deep probabilistic CCA for joint modeling of paired biomedical observations (2019)</p></li>
<li><p>Keywords: PCCA, deep learning, paired biomedical observations</p></li>
<li><p>We hypothesize that end-to-end training will learn features that are good for both reconstruction and joint modeling.</p></li>
<li><p>“interpretability” is a broad concept [23], here we mean specifically that we can identify small subsets of genes that are associated with specific image features.</p></li>
<li><p>Being Bayesian, this means we want a sparse prior. :)</p></li>
<li><p>Our scientific aim is to use this generative model on a number of downstream tasks such as annotating histology slides with predicted gene expression levels and analyzing clusters that capture shared structure in our paired data.</p></li>
</ul>
</div>
<div id="imagecca" class="section level3">
<h3><code>ImageCCA</code></h3>
<ul>
<li><p>Paper name: Joint analysis of gene expression levels and histological images identifies genes associated with tissue morphology (2019)</p></li>
<li><p>Use convolutional autoencoders and sparse canonical correlation analysis (CCA).</p></li>
<li><p>Suggests that genetic variation regulates population variation in tissue morphological traits.</p></li>
</ul>
</div>
</div>
<div id="scrna-seq" class="section level2">
<h2>scRNA-seq</h2>
<div id="netnmf-sc" class="section level3">
<h3><code>netNMF-sc</code></h3>
<ul>
<li><p>Paper name: netNMF-sc: Leveraging gene-gene interactions for imputation and dimensionality reduction in single-cell expression analysis (2019)</p></li>
<li><p>Keywords: Single-cell RNA seq, NMF, regularized network</p></li>
<li><p><code>netNMF-sc</code> combines network-regularized non-negative matrix factorization with a procedure for handling zero inflation in transcript count matrices.</p></li>
</ul>
</div>
<div id="scgene-fit" class="section level3">
<h3><code>scGene-Fit</code></h3>
<ul>
<li><p>Paper name: Optimal marker gene selection for cell type discrimination in single cell analyses (2019)</p></li>
<li><p>Keywords: cell type discovery</p></li>
<li><p>Goal: to select a small set of informative markers to identify and differentiate specific cell types or cell states as precisely as possible.</p></li>
<li><p>Uses label-aware compressive classification methods.</p></li>
</ul>
</div>
<div id="gt-ts" class="section level3">
<h3><code>GT-TS</code> :&gt;</h3>
<ul>
<li><p>Paper name: GT-TS: Experimental design for maximizing cell type discovery in single-cell data (2018)</p></li>
<li><p>Keywords: cell type discovery</p></li>
<li><p>GT-TS estimates how many cells are required for sampling from each tissue with the goal of maximizing cell type discovery across samples from multiple iterations.</p></li>
</ul>
</div>
<div id="tgplvm" class="section level3">
<h3><code>tGPLVM</code> :&gt;</h3>
<ul>
<li><p>Paper name: A robust nonlinear low-dimensional manifold for single cell RNA-seq data (2018)</p></li>
<li><p>Keywords: cell type discovery, dimension reduction, Manifold Learning, Gaussian Process Latent Variable Model</p></li>
</ul>
</div>
<div id="biisq" class="section level3">
<h3><code>BIISQ</code> :&gt;</h3>
<ul>
<li><p>Paper name: Bayesian nonparametric discovery of isoforms and individual specific quantification (2018 Nature Communications)</p></li>
<li><p>Keywords: isoform discovery from short-read RNA seq dara, stochastic variational inference</p></li>
</ul>
</div>
</div>
<div id="associationqt-studies" class="section level2">
<h2>Association/QT studies</h2>
<div id="bth" class="section level3">
<h3><code>BTH</code> :&gt;</h3>
<ul>
<li><p>Paper name: Statistical tests for detecting variance effects in quantitative trait studies (2018 Bioinformatics)</p></li>
<li><p>Keywords: Bayesian heteroskedastic linear regression model, identify variance effect of the quantitative trait</p></li>
</ul>
</div>
<div id="dimension-reduction" class="section level3">
<h3>Dimension reduction :&gt;</h3>
<ul>
<li><p>Paper name: Adaptive Randomized Dimension Reduction on Massive Data (2017)</p></li>
<li><p>Keywords: randomized dimension reduction, PCA, linear mixed models</p></li>
</ul>
</div>
</div>
<div id="network-inference" class="section level2">
<h2>Network inference</h2>
<div id="bets" class="section level3">
<h3><code>BETS</code></h3>
<ul>
<li><p>Paper name: Causal Network Inference from Gene Transcriptional Time Series Response to Glucocorticoids (2019)</p></li>
<li><p>Keywords: network inference</p></li>
<li><p>BETS uses elastic net regression and stability selection from bootstrapped samples to infer causal relationships among genes.</p></li>
<li><p>Transcriptional time series data (GTEx v6).</p></li>
</ul>
</div>
</div>
<div id="reinforcement-learning" class="section level2">
<h2>Reinforcement learning</h2>
<div id="reinforcement-learning-for-icu-data" class="section level3">
<h3>Reinforcement learning for ICU data</h3>
<ul>
<li><p>Paper name: An Optimal Policy for Patient Laboratory Tests in Intensive Care Units (2019)</p></li>
<li><p>Keywords: Reinforcement Learning, Dynamic Treatment Regimes, Pareto Learning</p></li>
</ul>
</div>
<div id="rl-with-admissible-rewards" class="section level3">
<h3>RL with admissible rewards</h3>
<ul>
<li><p>Paper name: Defining Admissible Rewards for High Confidence Policy Evaluation (2019)</p></li>
<li><p>Goal:</p>
<ul>
<li><p>develop a method to identify an admissible set of reward functions.</p></li>
<li><p>to learn a policy for weaning patients from mechanical ventilation.</p></li>
</ul></li>
</ul>
</div>
</div>
<div id="gaussian-process" class="section level2">
<h2>Gaussian Process</h2>
<div id="gp-lfm" class="section level3">
<h3><code>GP-LFM</code></h3>
<ul>
<li><p>Paper name: Patient-Specific Effects of Medication Using Latent Force Models with Gaussian Processes (2019)</p></li>
<li><p>Goal: model the physiological states of patients in medical time series data.</p></li>
<li><p>Uses: Gaussian process and latent force model</p></li>
<li><p>Application/problem: recover patient-specific response to the administration of three common drugs: one antihypertensive drug and two anticoagulants.</p></li>
</ul>
</div>
<div id="medgp" class="section level3">
<h3><code>MedGP</code></h3>
<ul>
<li><p>Paper name: Sparse Multi-Output Gaussian Processes for Medical Time Series Prediction (2018)</p></li>
<li><p>Keywords: ICU data, Gaussian process regression, online prediction</p></li>
</ul>
</div>
<div id="dpgp" class="section level3">
<h3><code>DPGP</code></h3>
<ul>
<li><p>Paper name: Clustering gene expression time series data using an infinite Gaussian process mixture model (2018)</p></li>
<li><p>Keywords: clustering genes, Dirichlet processes, Gaussian processes</p></li>
</ul>
</div>
</div>
<div id="variational-inference-related" class="section level2">
<h2>Variational inference related</h2>
<div id="meld" class="section level3">
<h3><code>MELD</code> :&gt;</h3>
<ul>
<li><p>Paper name: Fast Moment Estimation for Generalized Latent Dirichlet Models (2017)</p></li>
<li><p>Keywords: latent Dirichlet, generalized method of moments, compare methods with EM, variational inference, and MCMC</p></li>
</ul>
</div>
<div id="dcpf" class="section level3">
<h3><code>DCPF</code></h3>
<ul>
<li><p>Paper name: Dynamic Collaborative Filtering With Compound Poisson Factorization (2017)</p></li>
<li><p>Keywords: ratings data, matrix factorization, variational inference</p></li>
</ul>
</div>
</div>
<div id="neuroscience" class="section level2">
<h2>Neuroscience</h2>
<div id="fmri-analysis" class="section level3">
<h3>fMRI analysis :&gt;</h3>
<ul>
<li>Paper name: Computational approaches to fMRI analysis (2017)</li>
</ul>
</div>
</div>
<div id="main-problems-want-to-solve" class="section level2">
<h2>Main problems want to solve</h2>
<ul>
<li><ol style="list-style-type: decimal">
<li>Associate histological image phenotypes with high-dimensional genomic markers.</li>
</ol>
<ul>
<li><p>Image features are hard to identify.</p></li>
<li><p>Confounders are hard to control in high-dimensional data.</p></li>
</ul></li>
<li><ol start="2" style="list-style-type: decimal">
<li>Cell type discovery/ Single cell RNA seq analysis</li>
</ol></li>
</ul>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.0 (2019-04-26)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.4

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] workflowr_1.4.0 Rcpp_1.0.1      digest_0.6.19   rprojroot_1.3-2
 [5] backports_1.1.4 git2r_0.25.2    magrittr_1.5    evaluate_0.14  
 [9] stringi_1.4.3   fs_1.3.1        whisker_0.3-2   rmarkdown_1.13 
[13] tools_3.6.0     stringr_1.4.0   glue_1.3.1      xfun_0.7       
[17] yaml_2.2.0      compiler_3.6.0  htmltools_0.3.6 knitr_1.23     </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
